docs/docs/userguide/configuration/CONFIGURATION.md

```diff
@@ -134,9 +134,9 @@ The instance type must be a valid [AWS instance type](https://aws.amazon.com/ec2
 
     The instance type cannot be changed once the infrastructure has been provisioned.
 
-### EKS node count
+### Initial EKS node count
```

Comment:
I am thinking if we need to expose this variable at all? Why don't we start with one node in background and autoscaler will do the job to scale it up if is needed. I think this does not make sense, because after provisioning EKS, it takes time to products get installed and scaled, and autoscaler probably will scale down the nodes to 1 anyhow, regardless of this setting. 

Also, when the environment is alive, if user change this number, again it returns back to the minimum number of possible nodes by autoscaler again, so this variable is pointless.

Reply:
We may should change this variable to maximum number of the nodes in EKS, to limit the scaling (or min and max nodes). In this case, EKS will not grow unexpectedly even if an application scale up unwanted unlimited (by mistake or attack). And if add min, we will able to empower the user to reserve the minimum resources they need to save initial upscaling time.

Reply:
Isn't there a real requirement for being able to define the initial size of your cluster by more than 1 node? I see this requirement and the job of the autoscaler's as 2 separate needs.

Reply:
As I mentioned before, we can introduce lower range for node numbers (min number) defaulted `1` to start with the cluster with predefined nodes for those who want to deploy an enterprise size environment, or have multiple product in one cluster.